Riot/Modules/MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -2115,6 +2115,7 @@ - (void)sendLocationWithLatitude:(double)latitude
                         description:description
                            threadId:self.threadId
                           localEcho:&localEchoEvent
+                          assetType:MXEventAssetTypeGeneric
```

Comment:
Not sure where this broke, probably some WIP on develop of iOS or SDK. Adding here just so I can make a TF build and will remove before merging